src/orquestra/sdk/_base/_dsl.py

```diff
@@ -480,6 +486,40 @@ def __call__(self, *args: _P.args, **kwargs: _P.kwargs) -> _R:
             ),
         )
 
+    def resolve_task_source_data(
+        self, wf_default_source_import: Optional[Import] = None
+    ):
+        # if user set source import explicitly, do nothing
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏This comment says "if X do Y" but the if statement is "if not X do Z" could you expand the comment with both scenarios?